test/DemaConsulting.DotnetToolWrapper.Tests/DotnetToolWrapperTests.cs

```diff
@@ -0,0 +1,302 @@
+using System.Runtime.InteropServices;
+using System.Text.Json;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace DemaConsulting.DotnetToolWrapper.Tests;
+
+/// <summary>
+/// Tests for the DotnetToolWrapper application
+/// </summary>
+[TestClass]
+public class DotnetToolWrapperTests
```

Comment:
Please separate this one test file into two different files:
- One for unit testing the Program.cs methods
- One for running the DotnetToolWrapper DLL